app/main/index.ts

```diff
@@ -366,7 +366,9 @@ app.on('ready', () => {
 
 app.on('before-quit', () => {
 	mainWindow.webContents.send('save-domains');
-	isQuitting = true;
+	setTimeout(() => {
+		isQuitting = true;
```

Comment:
Hmm, how did you come to this conclusion here? This seems a workaround than a correct fix.